README.md

```diff
@@ -1,39 +1,28 @@
-Welcome to Intrinsic Physiology Feature Extractor (ipfx)
-========================================================
+Welcome to Intrinsic Physiology Feature Extractor (IPFX)
+==========================================
 
-ipfx is a python 2/3 package for computing intrinsic cell features from electrophysiology data.  This includes:
+IPFX is a Python package for computing intrinsic cell features from electrophysiology data. With this package you can:
 
-    * action potential detection (e.g. threshold time and voltage)
-    * cell quality control (e.g. resting potential stability)
-    * stimulus-specific cell features (e.g. input resistance)
+- Perform cell data quality control (e.g. resting potential stability)
+- Detect action potentials and their features (e.g. threshold time and voltage)
+- Calculate features of spike trains (e.g., adaptation index)
+- Calculate stimulus-specific cell features
 
 This software is designed for use in the Allen Institute for Brain Science electrophysiology data processing pipeline.
 
-## Quickstart:
+For usage and installation instructions, see the [documentation](https://ipfx.readthedocs.io/en/latest//).
 
-To run:
-
-```bash
- $ cd ipfx/ipfx/bin
- $ python pipeline_from_nwb.py input_nwb_file
-```
-User must specify the OUTPUT_DIR inside the pipeline_from_nwb.py
-
-Input:
-* input_nwb_file: a full path to the NWB file with cell ephys recordings
-
-Output:
-
- * pipeline_input.json: input parameters
- * pipeline_output.json: output including cell features
- * output.nwb: NWB file including spike times
- * log.txt: run log
- * qc_figs: index.html includes cell figures and feature table and sweep.html includes sweep figures
+Quick Start
+------------
+To get started check out the quick tutorial [quick_start](docs/quick_start.rst) 
```

Comment:
I would change to "To get started with analyzing data check the quick start instructions" with the link.

Reply:
maybe even something a bit conversational, like:
"To start analyzing data now, check out the <quick start instructions>. For a more in depth guide to IPFX, see <the tutorial>"

Reply:
added tutorial link as suggested, thanks.